ivy/container/general.py

```diff
@@ -230,6 +230,160 @@ def all_equal(
             prune_unapplied,
             map_sequences,
         )
+    
+    @staticmethod
+    def static_gather(
+        params: Union[ivy.Array, ivy.NativeArray, ivy.Container],
+        indices: Union[ivy.Array, ivy.NativeArray, ivy.Container],
+        axis: int = -1,
+        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+        to_apply: bool = True,
+        prune_unapplied: bool = False,
+        map_sequences: bool = False,
+        *,
+        out: Optional[ivy.Container] = None,
+    ) -> ivy.Container:
+        """
+        ivy.Container static method variant of ivy.gather. This method simply wraps
+        the function, and so the docstring for ivy.gather also applies to this
+        method with minimal changes.
+
+        Parameters
+        ----------
+        params
+            The container from which to gather values.
+        indices
+            Index array or container.
+        axis
+            optional int, the axis from which to gather from. Default is -1.
+        key_chains
+            The key-chains to apply or not apply the method to. Default is None.
+        to_apply
+            If True, the method will be applied to key_chains, otherwise key_chains
+            will be skipped. Default is True.
+        prune_unapplied
+            Whether to prune key_chains for which the function was not applied.
+            Default is False.
+        map_sequences
+            Whether to also map method to sequences (lists, tuples). Default is False.
+        device
+            device on which to create the array 'cuda:0', 'cuda:1', 'cpu' etc. Same as
+            ``x`` if None.
+        out
+            optional output container, for writing the result to.
+
+        ret
+            New container with the values gathered at the specified indices along the specified
+            axis.
+
+        Both the description and the type hints above assumes an array input for simplicity,
+        but this function is *nestable*, and therefore also accepts :code:`ivy.Container`
+        instances in place of any of the arguments.
+
+        Examples
+        -------------------
```

Comment:
Please edit this to match the length of Examples

Reply:
fixed in 79c7d2eda0d2320b133686ed9e5b0164356c8412